app/controllers/open_marc_records_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class OpenMarcRecordsController < ApplicationController
+  def index
+    @data_dumps = OpenMarcRecord.data_dumps.map.with_index.to_h
```

Comment:
Would it make sense to have all the enumerable processing together in the `#data_dumps` method in the model?

Reply:
I'm mapping the array to a hash for display in the view. I don't use the the data in this format anywhere else, which is why I didn't move it into the method.

Reply:
Would it be good to make this a 1-based index, so that the number in the index and the number in the filename match?

Reply:
I don't think it really matters, the index value isn't related to the number in the filename and the naming of the files could change in the future.